topics.asciidoc

```diff
@@ -54,6 +56,7 @@ Review of some mechanics and terms from Mastering Bitcoin. This would probably b
 * Scale Payments (amount of possible payments independent of block size / data base size)
 * increase privacy (not all payments stored on chain)
 * decrease settlement time (get rid of confirmations)
+* need of payment receipt (normally lacking in a Bitcoin mainchain transaction)
```

Comment:
Do you know the routing section of my slide deck? I introduce the invoice based system to explain how it is currently needed for the routing layer. While a getting started section similar to Mastering Bitcoin will certainly explain the payment process I am not sure if we need it in the motivation chapter. In general I agree that bolt 11 and invoices / receipts will need special care

Reply:
Yeah, I was torn when including the receipt point in that section. I now agree with your reasoning that payment receipt was not the *motivation* behind Lightning Network, but more of an added benefit that it provides. You can probably explain the importance of receipt if you include a section on spontaneous payments.

Reply:
I pushed a commit, so that it is easier for you to directly merge.

Reply:
Also added channel factories in Bolt 1.1/2.0 section